Add instructions for installing optional dependencies (#3506)
GenericMappingTools/pygmt: doc/install.md

For the minimum supported versions of the dependencies, please see [](minversions.md).
:::

:::{note}

> Some optional dependencies adds more functionality to PyGMT. For a complete list of
> optional dependencies, refer to [](ecosystem.md).

Reviewer: ```suggestion
Some optional dependencies add more functionality to PyGMT. For a complete list of these
optional dependencies, refer to [](ecosystem.md).
```

From now on, all commands will take place inside the virtual environment called `pygmt`
and won't affect your default `base` installation.

::::: {tip}

> You can also install PyGMT's optional dependencies for more functionalities.

Reviewer: ```suggestion
You can also install PyGMT's optional dependencies for more functionality.
```

From now on, all commands will take place inside the virtual environment called `pygmt`
and won't affect your default `base` installation.

::::: {tip}

> You can also enable more PyGMT's functionality by installing PyGMT's optional dependencies in the environment.

Reviewer 2: ```suggestion
You can also enable more PyGMT functionality by installing PyGMT's optional dependencies in the environment.
```

From now on, all commands will take place inside the virtual environment called `pygmt`
and won't affect your default `base` installation.

::::: {tip}

> You can also install PyGMT's optional dependencies for more functionality (after
> activating the environment).

Author: ```suggestion
You can also enable more PyGMT's functionality by installing PyGMT's optional dependencies in the environment.
```
